FundParser/BL/Exceptions/ApiErrorException.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FundParser.BL.Exceptions
+{
+    public class ApiErrorException : Exception
+    {
+        public ApiErrorException(string? message) : base(message) { }
```

Comment:
Do we need this nullable? Not exactly an issue, but it compiles without it and doesn't seem to produce any warnings either